db/db_impl/db_impl_write.cc

```diff
@@ -1124,13 +1124,29 @@ IOStatus DBImpl::WriteToWAL(const WriteThread::WriteGroup& write_group,
     //    writer thread, so no one will push to logs_,
     //  - as long as other threads don't modify it, it's safe to read
     //    from std::deque from multiple threads concurrently.
+    //
+    // Sync operation should work with locked log_write_mutex_, because:
```

Comment:
Thanks for the comment here. Just realized that the comment in line 1120 and so on will need to be updated due to this PR. Could you update it?

Reply:
Thanks for review.
Line 1120 describes why it is not necessary to add _mutex when accessing _logs. That description is appropriate. 
This root cause of this issue is file->Sync running without locked log_write_mutex_, not accessing _logs.
So I did not modify the comment on line 1120